### DISCUSSION TITLE
Airports with class national_park/state_park

### DISCUSSION THREAD

**Discussion by ksendzo** [OvertureMaps/data#245] [Discussion]
There are airports that have class *national_park*/*state_park*. 

Examples: 
- *Grand Canyon National Park Airport*
- *Hwange National Park Airport*

Here is the list of all the ids : *08b9750816af0fff0002c6b5cd4ed0aa, 08b26686e374cfff0002bdb24a0e23f7, 08b2641249d41fff0002be0c47159f39, 08ba735821a68fff0002bb13e894c5ef, 08b7a6a594a34fff0002b1e14ef857fb, 08b26a5b72188fff0002b34a92b336b0, 08b2994c74941fff0002bf0397034d12*.

**Comment by jenningsanderson** [comment]
Hey @ksendzo — thanks for pointing this out. 

These features are also present as airports in the base/infrastructure theme/type. 

As of the October release, these are no longer classified as national / state parks.
